eclass/aspell-dict-r1.eclass

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+# @ECLASS: aspell-dict-r1.eclass
+# @MAINTAINER:
+# maintainer-needed@gentoo.org
+# @AUTHOR:
+# Original author: Seemant Kulleen
+# @BLURB: An eclass to streamline the construction of ebuilds for new aspell dicts
+# @DESCRIPTION:
+# The aspell-dict-r1 eclass is designed to streamline the construction of
+# ebuilds for the new aspell dictionaries (from gnu.org) which support
+# aspell-0.50. Support for aspell-0.60 has been added by Sergey Ulanov.
+
+# @ECLASS-VARIABLE: ASPELL_LANG
+# @REQUIRED
+# @DESCRIPTION:
+# Which language is the dictionary for? It's used for the DESCRIPTION of the
+# package.
+
+# @ECLASS-VARIABLE: ASPOSTFIX
+# @REQUIRED
+# @DESCRIPTION:
+# What major version of aspell is this dictionary for?
+
+case "${EAPI:-0}" in
+	[0-5])
+		die "aspell-dict-r1.eclass is banned in EAPI ${EAPI:-0}"
+		;;
+	6)
+		;;
+	*)
+		die "Unknown EAPI ${EAPI:-0}"
+		;;
+esac
+
+EXPORT_FUNCTIONS src_configure src_install
+
+if [[ ! ${_ASPELL_DICT_R1} ]]; then
+
+MY_P=${P%.*}-${PV##*.}
+MY_P=aspell${ASPOSTFIX}-${MY_P/aspell-/}
+SPELLANG=${PN/aspell-/}
+S="${WORKDIR}/${MY_P}"
+DESCRIPTION="${ASPELL_LANG} language dictionary for aspell"
+HOMEPAGE="http://aspell.net"
+SRC_URI="mirror://gnu/aspell/dict/${SPELLANG}/${MY_P}.tar.bz2"
+
+IUSE=""
+SLOT="0"
+
+if [[ ${ASPOSTFIX} == 6 ]]; then
+	RDEPEND=">=app-text/aspell-0.60"
+else
+	RDEPEND=">=app-text/aspell-0.50"
+fi
+DEPEND="${RDEPEND}"
+
+# @FUNCTION: aspell-dict-r1_src_configure
+# @DESCRIPTION:
+# The aspell-dict-r1 src_configure function which is exported.
+aspell-dict-r1_src_configure() {
+	./configure || die
```

Comment:
I presume this is non-autoconf. In that case, explanatory comment would be helpful.